src/client/settings/templates/settings-cache.html

```diff
@@ -14,7 +14,7 @@
 
         <md-input-container>
             <md-checkbox ng-model="settings.config.deleteTmpOnClose">
-                Clear Tmp Folder after closing app?
```

Comment:
I don't agree with this. Since `butter-desktop-angular` is a complete rewrite from zero, I think we should give to devs as much as flexibility they want and they need.

Since this project is really not ready to use by lambda users, please hack it, rerite it completely and we will care about translations later (we always find people to translate in their native languages, I am not afraid by that).

(that's only my opinion, if a third person could give his opinion that would be great)